corgi/collectors/brew.py

```diff
@@ -116,54 +116,72 @@ def _parse_remote_source_url(cls, url: str) -> str:
         path = parsed_url.path.removesuffix(".git")
         return f"{parsed_url.netloc}{path}"
 
+    @classmethod
+    def _bundled_or_golang(cls, component: str) -> Optional[str]:
+        # Process bundled deps only; account for typed golang deps of type:
+        # "golang(golang.org/x/crypto/acme)"
+        if component.startswith("bundled("):
+            c = component.removeprefix("bundled(")
+        elif component.startswith("golang("):
+            c = component
+        else:
+            return None
+        # Strip right parens
+        return c.replace(")", "")
+
+    @classmethod
+    def _check_maven_component(cls, component: str, version: str) -> Optional[tuple]:
```

Comment:
You may need an import like "from typing import Type" here to make mypy happy:
```suggestion
    def _check_maven_component(cls, component: str, version: str) -> Optional[tuple[Type, str, str]]:
```
Or maybe you need to change the return signature:
```suggestion
    def _check_maven_component(cls, component: str, version: str) -> tuple:
```

Reply:
I updated it to:

```
    def _check_maven_component(
        cls, component: str, version: str
    ) -> Optional[tuple[Component.Type, str, str]]
```